src/Searchlight/LinqExecutor.cs

```diff
@@ -114,23 +114,25 @@ private static Expression BuildOneExpression(ParameterExpression select, BaseCla
                         case OperationType.StartsWith:
                             return Expression.TryCatch(
                                 Expression.Call(field,
-                                    typeof(string).GetMethod("StartsWith", new Type[] {typeof(string)}), value),
+                                    typeof(string).GetMethod("StartsWith", new Type[] {typeof(string), typeof(StringComparison)}), 
+                                    value, Expression.Constant(StringComparison.CurrentCultureIgnoreCase)),
```

Comment:
@Peekoe @tspence If we don't care about cultures, we may want to do a find/replace globally with OrdinalIgnoreCase... it's faster than current culture.
https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings

Reply:
I'll work on that, thanks for the review!